jerry-core/include/jerryscript-core.h

```diff
@@ -754,12 +787,20 @@ void jerry_heap_free (void *mem_p, size_t size);
  */
 jerry_context_t *jerry_create_context (uint32_t heap_size, jerry_context_alloc_t alloc, void *cb_data_p);
 
+/**
+ * Backtrace functions.
+ */
+jerry_value_t jerry_get_backtrace (uint32_t max_depth);
+void jerry_capture_backtrace (jerry_backtrace_callback_t callback, void *user_p);
```

Comment:
Most of the backtrace functions start with the `jerry_backtrace` prefix. Could we rename this to `jerry_backtrace_capture`? Sidenote: "capture" seems a bit off for me because the engine already knows the backtrace we just access/read it here, but at the moment I don't know any good alternatives for the name.

Reply:
I think capture term is used in connection with backtrace frequently